newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackPurchaseDialog.js

```diff
@@ -163,6 +163,15 @@ const PrivateAssetPackPurchaseDialog = ({
     else onStartPurchase();
   };
 
+  React.useEffect(
+    () => {
+      onWillPurchase();
+    },
+    // Launch the start process directly when the dialog is opened, to avoid an extra click.
```

Comment:
Do we want this though? this dialog is kind of a safe reminder that it will be linked to the account "xxx@yyy"

Reply:
I think this can be the role of the web page giving the choice between Stripe and Paypal. It could even display again the thumbnail.
Otherwise, we go from:

Buy button click -> confirmation -> web page opens -> enter your credit card info -> done!

to
Buy button click -> confirmation -> web page opens -> choose paypal or stripe -> enter your credit card info -> done!

And I think it's already too much. Maybe a good design would tell that Stripe/Paypal buttons should be directly the "Buy buttons".